### PR TITLE
.gitignore: Ignore dynamic ui components

### PR DESCRIPTION
**Pull request by danrue**
Signed-off-by: Dan Rue <dan.rue@linaro.org>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -57,3 +57,5 @@ ENV/
 node_modules
 
 /app/admin/ui/public
+mr_provisioner/admin/ui/package-lock.json
```
Comment (bwalex) [inline .gitignore]: It'd be better to actually generate this file and add it under version control (but making sure it's perfectly in sync with the yarn lock file). https://github.com/yarnpkg/yarn/issues/3614 is relevant.

Comment (bwalex) [reply .gitignore]: In the meantime, if you don't plan on doing that right now, I'd rather this didn't go into the .gitignore.

**Comment by danrue** [comment]
Fixed and rebased.

**Comment by danrue** [comment]
Bump

**Comment by bwalex** [comment]
It's not ideal that the npm and yarn lock files are not in sync, but LGTM anyway.

**Comment by danrue** [comment]
I switched to yarn after grief with npm and this file - so I don't really care either. If you want me to drop it, I can. Otherwise, please merge?

**Comment by gemagomez** [comment]
Fixed in b7cfd4671537c229711005885786c15102347b7d and b36844cc22077836ff1055fbb5a1718c830f2a75.

**Pull request closed without merging**